tests/system/action/motion/test_create.py

```diff
@@ -340,3 +352,111 @@ def test_create_with_submitters(self) -> None:
         assert submitter_2.get("meeting_id") == 222
         assert submitter_2.get("user_id") == 57
         assert submitter_2.get("motion_id") == 1
+
+    def test_create_no_permission(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion.create",
+            {
+                "title": "test_Xcdfgee",
+                "meeting_id": 1,
+                "workflow_id": 12,
+                "text": "test",
+            },
+        )
+
+    def test_create_permission_simple_fields(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion.create",
+            {
+                "title": "test_Xcdfgee",
+                "meeting_id": 1,
+                "workflow_id": 12,
+                "text": "test",
+            },
+            Permissions.Motion.CAN_CREATE,
+        )
+
+    def test_create_permission_missing_can_manage(self) -> None:
+        self.create_meeting()
+        self.user_id = self.create_user("user")
+        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [3])
+        self.set_group_permissions(3, [Permissions.Motion.CAN_CREATE])
+        self.set_models(self.permission_test_model)
+        response = self.request(
+            "motion.create",
+            {
+                "title": "test_Xcdfgee",
+                "number": "X13",
+                "meeting_id": 1,
+                "workflow_id": 12,
+                "text": "test",
+            },
+        )
+        self.assert_status_code(response, 403)
+        assert "Forbidden fields:" in response.json["message"]
```

Comment:
plase assert the field names, that are forbidden.

Reply:
Okay.